testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -11,11 +11,15 @@ package testeng
 //     emails: List of emails that should be notified when job fails, e.g.: ['email1', 'email2']
 // ]
 
+// Cron values for: midnight-7:59am ET, Sunday-Thursday (times are in UTC)
+// See https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.jelly
+def cronOffHoursBusinessWeek = '* H(4-11) * * H(0-4)'
```

Comment:
I believe the cron value should be `H H(4-11) * * H(0-4)`, currently it's running the job every minute for that random hour.
<img width="353" alt="Screen Shot 2020-03-25 at 2 08 11 PM" src="https://user-images.githubusercontent.com/2851134/77523028-f3ac4000-6ea6-11ea-9666-71879cf276d4.png">

Reply:
Made a fix for this https://github.com/edx/jenkins-job-dsl/pull/933.

Reply:
Thanks again @waheedahmed.